inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/utils/HttpContextUtils.java

```diff
@@ -104,7 +104,10 @@ public static String getBodyString(ServletRequest request) {
                 }
             }
         } catch (IOException e) {
-            log.error("failed to get body string of request={}", request);
+            log.error("failed to get body string of request={}", request, e);
+        }
+        if (sb.length() == 0) {
```

Comment:
1. `sb` -> `builder`
2. Please extract a constant for `{}`, such as `EMPTY_BODY`.

Reply:
fixed, thx